app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+# Helper methods for session handling
+module SessionsHelper
+  def log_in(token)
+    cookies.permanent[:remember_token] = token
+  end
+
+  def log_out
```

Comment:
This should be in the Application controller.

Reply:
that just deletes the token from the client side. It's fine where it is.